### PR TITLE
Fixes https://github.com/lenz99-/lme4cens/issues/4

### PR DESCRIPTION
**Pull request by moosterwegel**
Hi there,

Thanks for creating this package. I believe this pull requests fixes https://github.com/lenz99-/lme4cens/issues/4 

Cheers,

Max

### CODE REVIEW

**Comment by lenz99** [comment]
Thanks for reporting this issue and proposing a fix. I am looking into it.
-m

File: R/zzz.R [hunk 1]
```diff
@@ -62,7 +62,7 @@ ranef.lmercens <- function(obj){
   respMatrix <- as.matrix(obj$ingredients$fr[[1L]])
   stopifnot( "status" %in% colnames(respMatrix) )
   obsInd <- respMatrix[,"status", drop=TRUE] == 1L  ## eventually could use also interval-censored (=> take mean of interval)
-  fixef_resids <-  respMatrix[obsInd, 1L, drop=FALSE] - obj$ingredients$X[obsInd,] %*% fixef(obj)
+  fixef_resids <-  respMatrix[obsInd, 1L, drop=FALSE] - drop(obj$ingredients$X[obsInd,] %*% matrix(fixef(obj)))
```
Comment (lenz99) [inline R/zzz.R:65]: `obj$ingredients$X[obsInd,, drop=FALSE]` does the trick already.
It enforces that this first term remains a matrix and `%*%` will work.

**Review by lenz99 — CHANGES_REQUESTED** [review]
Can you confirm that `obj$ingredients$X[obsInd,, drop=FALSE]` fixes the bug with models with models that have only an intercept term as fixed effect?

If yes, please adapt your PR and I will merge it..
-m

**Comment by moosterwegel** [comment]
Hi Matthias,
Thanks for the quick reply. Your solution indeed also works. I have adapted my PR.
Max

**Review by ghost — APPROVED** [review]

**Pull request merged**